src/VideoPlayingScreen/PlayVideo.css

```diff
@@ -155,9 +158,10 @@
   }
 }
 /* model */
+
 .model {
   border: 1px solid black;
-  z-index: 1;
+  z-index: 111111;
   position: absolute;
   background-color: #212121;

```

Comment:
We can use a single file which contains all the colors as variables and then it would be easier to make changes